AutomatedTesting/Gem/PythonTests/scripting/ScriptEvent_AddRemoveParameter_ActionsSuccessful.py

```diff
@@ -49,11 +49,11 @@ class ScriptEvent_AddRemoveParameter_ActionsSuccessful:
     """
 
     def __init__(self):
-        editor_window = None
-        asset_editor = None
-        asset_editor_widget = None
-        asset_editor_row_container = None
-        asset_editor_menu_bar = None
+        self.editor_main_window = None
+        self.asset_editor = None
+        self.asset_editor_widget = None
+        self.asset_editor_row_container = None
+        self.asset_editor_menu_bar = None
 
     def create_script_event(self) -> None:
         action = pyside_utils.find_child_by_pattern(self.asset_editor_menu_bar, {"type": QtWidgets.QAction, "text": SCRIPT_EVENT_UI})
```

Comment:
This looks like a good function to extract into scripting tools.

As well as the other two functions here. Since we're going back and refactoring again, all of these should be considered too.

Reply:
moved these functions over to the tools file. The saving behavior was already in the tools file so I used that for saving and retrieving the save result so we can verify it's saved successfully.